dhall-json/src/Dhall/JSON.hs

```diff
@@ -951,13 +953,18 @@ handleSpecialDoubles specialDoubleMode =
 codeToValue
   :: Conversion
   -> SpecialDoubleMode
-  -> Text  -- ^ Describe the input for the sake of error location.
+  -> Maybe FilePath  -- ^ The source file path. If no path is given, imports
+                     -- are resolved relative to the current directory.
```

Comment:
@deepfire You introduced the `Text` input description in https://github.com/dhall-lang/dhall-json/pull/20. Is this change ok for you?

Reply:
Yeah, this change is fine